[BUG] E2E test container is failed python install

**Describe the bug**
- The pyenv installer uses `aria2c`, `curl`, or `wget`.
- But Those is not installed in the E2E test container, failed python install.
```
Downloading Python-3.8.7.tar.xz...
-> https://www.python.org/ftp/python/3.8.7/Python-3.8.7.tar.xz
error: failed to download Python-3.8.7.tar.xz

BUILD FAILED (Ubuntu 20.04 using python-build 2.2.2-1-gf2925393)

Results logged to /tmp/python-build.20211126065120.68.log

Last 10 log lines:
/tmp/python-build.20211126065120.68 /
error: please install `aria2c`, `curl`, or `wget` and try again
```

**Additional context**
- Install `aria2c`, `curl`, or `wget` in the container.

--- a/tests/Dockerfile
+++ b/tests/Dockerfile
@@ -29,6 +29,7 @@ RUN apt-get update -q \
  language-pack-ja-base \
  language-pack-ja \
  git \
+ curl \
  libyaml-cpp-dev
 
 # remove unnessesory package files

